apps/verticalhome/manifest.webapp

```diff
@@ -15,6 +15,7 @@
     "storage": {},
     "webapps-manage": {},
     "systemXHR": {},
+    "alarms": {},
```

Comment:
What is this going to be for?  I haven't see any code using this permission yet.  Are you sure you want to add this now?